src/locales/default.json

```diff
@@ -5,6 +5,8 @@
   "backToHome": "Home",
   "actions": "Actions",
   "results": "Results",
+  "resultsError": "Results could not be processed. This is most likely due to a malconfigured strategy or a performance issue with an involved contract. Please make sure none of the above applies before contacting support.",
+  "contactSupport": "Contact support",
```

Comment:
```suggestion
  "contactSupport": "Get help",
```

Reply:
Contact support sounds a bit like we are a company providing professional support.

Reply:
Changed it but... Why can't a community project have professional support?